<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainToolBar {
border: none;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);
}
/*
#menuBar {
border: none;
color: white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);
}

#menuBar QWidget {
border: none;
color: white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);
}*/
QHeaderView::section {
border: none;
color: white;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);
 }


#centralWidget {
border: none;
background: white;
}

#mainToolBar QToolButton {
color: #333;
border: 2px solid #555;
border-radius: 5px;
padding: 1px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 10px;
}

#mainToolBar QToolButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

#mainToolBar QToolButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}
#mainToolBar QToolButton:checked {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionScreen_shot"/>
    <addaction name="separator"/>
    <addaction name="actionGenerate_code"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionInsert_Line"/>
    <addaction name="actionUnlink"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_Node"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Graph</string>
    </property>
    <addaction name="addBaseGraph"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShow_Preview"/>
   </widget>
   <widget class="QMenu" name="menuScientific">
    <property name="title">
     <string>Scientific</string>
    </property>
    <addaction name="actionScientific_pictures"/>
    <addaction name="actionScript_Editor"/>
    <addaction name="actionRead_Log"/>
    <addaction name="actionAggrigate_Properies"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGraph"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuScientific"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionScript_Editor"/>
   <addaction name="separator"/>
   <addaction name="actionInsert_Line"/>
   <addaction name="actionUnlink"/>
   <addaction name="actionRemove_Node"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStep"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Preview"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
  </widget>
  <widget class="QDockWidget" name="preview">
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">

#preview QWidget {
border: none;
background: white;
}
</string>
   </property>
   <property name="windowTitle">
    <string>Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="previewlay">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionRemove_Node">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/118.png</normaloff>:/images/images/118.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionLink_Nodes">
   <property name="text">
    <string>Link Nodes</string>
   </property>
  </action>
  <action name="actionInsert_Line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/link.png</normaloff>:/images/images/link.png</iconset>
   </property>
   <property name="text">
    <string>Link Mode</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/119.png</normaloff>:/images/images/119.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/7.png</normaloff>:/images/images/7.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGenerate_code">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/2.png</normaloff>:/images/images/2.png</iconset>
   </property>
   <property name="text">
    <string>Generate code</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/98.png</normaloff>:/images/images/98.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/91.png</normaloff>:/images/images/91.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionUnlink">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/link_break.png</normaloff>:/images/images/link_break.png</iconset>
   </property>
   <property name="text">
    <string>Unlink</string>
   </property>
  </action>
  <action name="actionShow_Preview">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/27.png</normaloff>:/images/images/27.png</iconset>
   </property>
   <property name="text">
    <string>Show Preview</string>
   </property>
  </action>
  <action name="actionScreen_shot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/125.png</normaloff>:/images/images/125.png</iconset>
   </property>
   <property name="text">
    <string>Screen shot</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/reset.png</normaloff>:/images/images/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset all graphs</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="toolTip">
    <string>Create a new graph</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/7.png</normaloff>:/images/images/7.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save Graph with a new name</string>
   </property>
  </action>
  <action name="actionScientific_pictures">
   <property name="text">
    <string>Scientific pictures</string>
   </property>
  </action>
  <action name="actionRecent_files">
   <property name="text">
    <string>Recent files</string>
   </property>
  </action>
  <action name="actionScript_Editor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/script</normaloff>:/images/script</iconset>
   </property>
   <property name="text">
    <string>Script Editor</string>
   </property>
  </action>
  <action name="addBaseGraph">
   <property name="text">
    <string>Add Base Graph</string>
   </property>
  </action>
  <action name="actionRead_Log">
   <property name="text">
    <string>Read Log</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/step</normaloff>:/images/step</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>One frame forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionAggrigate_Properies">
   <property name="text">
    <string>Aggrigate Properies</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
